crates/uv-build-backend/src/lib.rs

```diff
@@ -539,3 +545,75 @@ mod tests {
         "###);
     }
 }
+
+/// Determine the source root and the module root.
+fn find_roots(
+    source_tree: &Path,
+    pyproject_toml: &PyProjectToml,
+    module_root: &Path,
+    module_name: Option<&Identifier>,
+) -> Result<(PathBuf, PathBuf), Error> {
```

Comment:
It might be good to tweak the names here or beef up the docs just a tad. For example, the docs (and names in the implementation) suggest this both returns and accepts a module root.